tests/components/notify/test_html5.py

```diff
@@ -97,18 +91,17 @@ def test_registering_new_device_view(self):
         """Test that the HTML view works."""
         hass = MagicMock()
 
-        with tempfile.NamedTemporaryFile() as fp:
-            hass.config.path.return_value = fp.name
-            fp.close()
+        m = mock_open()
+        with patch('homeassistant.components.notify.html5.open', m):
             service = html5.get_service(hass, {})
 
             assert service is not None
 
             # assert hass.called
-            assert len(hass.mock_calls) == 3
+            assert len(hass.mock_calls) == 4
```

Comment:
Since I removed the `hass.config.path.return_value = fp.name` lines from the code, hass.config.path() now returns a mock.  Then, on the call to `os.path.isfile` on [line 126 of `_load_config`](https://github.com/home-assistant/home-assistant/blob/d58548dd1c830f0aa5b6d68c8b9e0c68be5e014e/homeassistant/components/notify/html5.py#L126), results in an extra call to `hass`.  Whereas `mock_calls` once looked like this:
```python
 call.config.path('html5_push_registrations.conf'),
 call.wsgi.register_view(<homeassistant.components.notify.html5.HTML5PushRegistrationView object at 0x7fba746dc438>),
 call.wsgi.register_view(<homeassistant.components.notify.html5.HTML5PushCallbackView object at 0x7fba746df5c0>)
```
it now looks like this:
```python
 call.config.path('html5_push_registrations.conf'),
 call.config.path().__index__(),
 call.wsgi.register_view(<homeassistant.components.notify.html5.HTML5PushRegistrationView object at 0x7fba746dc438>),
 call.wsgi.register_view(<homeassistant.components.notify.html5.HTML5PushCallbackView object at 0x7fba746df5c0>)
```
so, the call to `os.path.isfile` with a mock object as an argument seems to trigger the extra `call.config.path().__index__()`.  I tried to replace `hass.config.path.return_value` with a string, but ran into a problem somewhere.  I can't remember where, though.  Let me take a closer look and get back to you.